package/MDAnalysis/core/groups.py

```diff
@@ -1182,17 +1214,63 @@ def select_atoms(self, sel, *othersel, **selgroups):
                 therefore have atoms that were initially absent from the
                 instance :meth:`~select_atoms` was called from.
 
+            global *selection*
```

Comment:
I might have misread this one: here *selection* is not the kw but the content place holder, isn't it? I can't remember if there's anything specific prescribed so then leave it as we had it. I thought that this one here was similar to https://github.com/MDAnalysis/mdanalysis/pull/1080#discussion_r89420562 ; in any case, that's only docs.